openfe/protocols/settings.py

```diff
@@ -0,0 +1,47 @@
+"""
```

Comment:
My intention was that the settings module would only be for settings objects that are (intended to be) re-used by multiple protocols, and that a protocol's other settings objects would be in that Protocol's own module - so for instance, `RelativeHybridTopologyProtocolSettings` is in `openfe.protocols.openmm_rfe`. So it might be more consistent to move `AlchemicalSamplerSettings` to that module if that's not really re-usable by other protocols - I had to make a guess.

Reply:
So right now, anything in openmm_utils is "reusable", but only within the scope of OpenMM protocols. (Anything in a specific protocol is local to that protocol) When we move to gmx I expect most these to not apply.